UnitTests/OIDTokenRequestTests.m

```diff
@@ -239,6 +239,10 @@ - (void)testSecureCoding {
   XCTAssertNotNil(request.additionalHeaders, @"");
   XCTAssertEqualObjects(request.additionalHeaders[kTestAdditionalHeaderKey],
                         kTestAdditionalHeaderValue, @"");
+  
```

Comment:
These tests are helpful to add, but they only test this class's [`NSSecureCoding`](https://developer.apple.com/documentation/foundation/nssecurecoding?language=objc) conformance.

I was thinking of also adding a new test case that is specifically focused on ensuring that the additional headers are  present in the token request.

```
- (void)testThatAdditionalHeadersAreInTokenRequest {
  // Construct an `OIDTokenRequest` with `additionalHeaders` 
  // Assert that the `NSURLRequest`'s headers contain what you expect
}
```

This sort of test can verify that the typo is fixed, and can also help to prevent similar typos in the future.

Reply:
@mdmathias 
Please check last commit.

Reply:
Hello, may I ask if there is anything I can do on my end to help with this issue?